src/app/components/files/files.component.html

```diff
@@ -80,7 +99,15 @@
           </data-columns>
         </adf-document-list>
 
-        <adf-pagination acaPagination [target]="documentList"> </adf-pagination>
+        <adf-pagination
+          acaPagination
```

Comment:
there's a special directive that applies a set of behaviors for multiple pages. Can you see if it's possible to move your custom code there? https://github.com/Alfresco/alfresco-content-app/blob/develop/src/app/directives/document-list.directive.ts#L40. Otherwise only "Personal Files" are affected, other pages won't have filters